server.js

```diff
@@ -1,9 +1,27 @@
-const express = require('express');
-const server = express();
+const http = require('http');
+const WebSocket = require('ws');
 const chalk = require('chalk');
 const constructXML = require('./src/junitFormatter');
 
-server.use(express.json());
+// Initialize a server
+const server = http.createServer();
+
+// Setup local variables for server
+server.locals = { appBooted: false};
```

Comment:
This works and we've kept the same `locals` from when we were in express land. I don't know whether there's a 'proper/expected way' to do this for our websocket server - @jalada all good?

Reply:
You could literally just have a global variable. `var appBooted = false;`